Meta/gn/secondary/Userland/Libraries/LibAudio/BUILD.gn

```diff
@@ -13,7 +13,6 @@ shared_library("LibAudio") {
     "PlaybackStream.cpp",
     "QOALoader.cpp",
     "QOATypes.cpp",
-    "RIFFTypes.cpp",
```

Comment:
FWIW if you wanted to update the gn build to work with these changes, you would need to add a new LibRIFF/BUILD.gn and add `//Userland/Libraries/LibRiff` as a dependency to LibAudio and LibGfx. But this smells like an IDE's "delete all references" sort of change :)